File: crushapp/lib/app/models/courseModel.dart
class CourseModel {
  int? courseId;
  int? courseCode;
  String? courseName;
  int? questions;
  String? courseUrl;

  CourseModel(
      {this.courseId,
      this.courseCode,
      this.courseName,
      this.questions,
      this.courseUrl});

  CourseModel.fromJson(Map<String, dynamic> json) {
    courseId = json['CourseId'];
    courseCode = json['CourseCode'];
    courseName = json['CourseName'];
    questions = json['Questions'];
    courseUrl = json['CourseUrl'];
  }
}

File: MdLabScience.WebAPI/Models/CourseModel.cs
using System;

namespace StudentCertificateManagement.Models
{
    public class CourseModel
    {
        public int CourseId { get; set; }
        public string CourseCode { get; set; }
        public string CourseName { get; set; }
        public bool IsActive { get; set; }
        public string CourseUrl { get; internal set; }
    }
}
